generators/build/templates/Gemfile

```diff
@@ -6,6 +6,4 @@ group :test do
   git 'https://github.com/Azure/terramodtest.git' do
     gem 'terramodtest', :tag => 'v0.1.0'
```

Comment:
Is the version updated ?

Reply:
It is the version in my own package. I think the version should be 'v0.1.5' or 'v0.2.0' here.